cmd/autogen.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -e VERSION ]; then
 	( cd .. && ./mkversion.sh )
 fi
 
-# Sanity check, are we in the right directory?
+# Validity check, are we in the right directory?
```

Comment:
I think in this kind of case and similar,  precodintion check would be a clearer/better term